script/1_0_2current_en_script/Common/00_00.txt

```diff
@@ -282,52 +282,52 @@ The woman walked away as if it was a matter of course.
 Tomorrow we can talk again, just like today.
 I was glad. For the first time since I woke up from the accident, I felt human again.
 <Page78>
-It became my routine to go to the field in the afternoon.
+It became routine for me to go to the field in the afternoon.
 The woman would get upset if I called her Aoko. She said she didn't like her name.
-After much pondering, I decided to call her "Sensei" because she seemed to be an important person.
+After much pondering, I decided to call her 'Sensei' because she seemed to be an important person.
 <Page79>
 Sensei listened to me seriously no matter what I said and solved my problems with just a few words.
 <Page80>
 After the accident, I had become uncomfortable talking to people, but thanks to her, I was able to gradually return to my normal self.
 <Page81>
-Only when I was in this field did my anxiety disappear because I knew that I could be my normal self here.
-Even though I didn't know who she was, she might really be a teacher in some school.
+Only when I was in this field did my anxiety disappear, because I knew that I could be my normal self here.
+Even though I didn't know who she was, she might really be a teacher at some school.
 <Page82>
-But that was just a "what if" that had nothing to do with me.
+But that was just a 'what if' that had nothing to do with me.
 I enjoyed being with Sensei.
-It was that simple, that was all that mattered.
+It was that simple, and that was all that mattered.
```

Comment:
Not sure this `and` is needed - "It was that simple, [being with Sensei] was all that mattered" is I think the intent here, and "It was that simple, _and_ being with Sensei" doesn't read right